src/main/java/com/lineate/testyourlexicon/repositories/LanguageRepository.java

```diff
@@ -0,0 +1,35 @@
+package com.lineate.testyourlexicon.repositories;
+
+import com.lineate.testyourlexicon.dto.SingleTranslationDto;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@RequiredArgsConstructor
+public class LanguageRepository {
```

Comment:
I would name this repository `TranslationRepository`. `LanguageRepository` sounds too abstract and also this repository works with translations table in the database